io/net/virt-net/network_virtualization.py

```diff
@@ -114,6 +114,10 @@ def setUp(self):
             self.cancel('Backing Device counts and vios name counts differ')
         self.backingdev_count = len(self.backing_adapter)
         self.bandwidth = self.params.get("bandwidth", '*', default=None)
+        self.vnic_priority = self.params.get("priority", '*', default=50).split(',')
+        if len(self.vnic_priority) != len(self.backing_adapter):
+            self.cancel('Backing Device counts and priority counts differ')
+        self.auto_failover = int(self.params.get("auto_failover", '*', default=1))
```

Comment:
It is better to evaluate this value after this, since user can give wrong values like 2 (example).
So, you can make it 0 if 0, and 1 if non-zero.

Reply:
I will have a check for that, so for 0 it is 0
for 1 and non zero or empty it is 1

Reply:
yes, perfect. I had missed empty scenario.

Reply:
done